ast.go

```diff
@@ -997,7 +1033,7 @@ func (node Identifier) IsEmpty() bool {
 // CreateTable represents a CREATE TABLE statement.
 type CreateTable struct {
 	Table       *Table
-	Columns     []*ColumnDef
+	ColumnsDef  []*ColumnDef
```

Comment:
This is just a change in the name to be more precise. `Columns` usually indicate a list of column names, and here we want to indicate a list of column definitions.